orc8r/cloud/go/mproto/mocks/mproto_mock.proto

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package magma.orc8r.mproto_mock;
+option go_package = "protos";
+
+// TestDataBasic is a protobuf message with basic field types for testing deterministic encoding
```

Comment:
Here and elsewhere: "is a protobuf message" is unnecessary as it's self-evident 🙂